sssom/validators.py

```diff
@@ -0,0 +1,89 @@
+import logging
+
+from jsonschema import ValidationError
+from linkml.validators.jsonschemavalidator import JsonSchemaDataValidator
+from sssom_schema import MappingSet
+
+from sssom.parsers import to_mapping_set_document
+from sssom.util import MappingSetDataFrame, get_prefix_from_curie
+
+from .constants import ENTITY_REFERENCE_SLOTS, SCHEMA_VIEW, SCHEMA_YAML
+
+
+def json_schema_validate(msdf: MappingSetDataFrame) -> bool:
```

Comment:
keep the json_schema_validate method but also add a generic

```
validate(input_path: str, validation_types = DEFAULT_VALIDATION_TYPES: list[str]) -> bool
```

In this method, add a switch over all the `validation_types`, and when say case `SchemaValidationType.JsonSchema` call the schema_validation method.